src/StdConstants.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
+
+import {IMulticall3} from "./interfaces/IMulticall3.sol";
+import {Vm} from "./Vm.sol";
+
+library StdConstants {
+    /// @dev Cheat code address.
+    /// Calculated as `address(uint160(uint256(keccak256("hevm cheat code"))))`.
+    Vm internal constant VM = Vm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D);
+    /// @dev console.sol and console2.sol work by executing a staticcall to this address.
+    /// Calculated as `address(uint160(uint88(bytes11("console.log"))))`.
+    address internal constant CONSOLE = 0x000000000000000000636F6e736F6c652e6c6f67;
+    /// @dev Used when deploying with create2.
+    /// Taken from https://github.com/Arachnid/deterministic-deployment-proxy.
+    address internal constant CREATE2_FACTORY = 0x4e59b44847b379578588920cA78FbF26c0B4956C;
+    /// @dev The default address for tx.origin and msg.sender.
+    /// Calculated as `address(uint160(uint256(keccak256("foundry default caller"))))`.
+    address internal constant DEFAULT_SENDER = 0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38;
```

Comment:
For equivalence we should also update `DEFAULT_SENDER` in `Base.sol` imo

https://github.com/foundry-rs/forge-std/blob/bf909b22fa55e244796dfa920c9639fdffa1c545/src/Base.sol#L14-L15

Confirming address is correct:

```
➜ address(uint160(uint256(keccak256("foundry default caller"))))
Type: address
└ Data: 0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38
```

Reply:
Done.